plugins/woocommerce/src/Admin/RemoteInboxNotifications/Transformers/ArrayFlatten.php

```diff
@@ -20,7 +20,11 @@ class ArrayFlatten implements TransformerInterface {
 	 *
 	 * @return mixed|null
 	 */
-	public function transform( $value, stdClass $arguments = null, $default = null ) {
+	public function transform( $value, stdClass $arguments = null, $default = array() ) {
```

Comment:
@ilyasfoo Thanks for pointing that out. I missed that for `ArrayColumn` and just updated it in c819b75df104c79ed6545e344ba50495bce44535 . I think we should keep the default value of `ArraySearch` null [because we are returning null when value is not found](https://github.com/woocommerce/woocommerce/blob/trunk/plugins/woocommerce/src/Admin/RemoteInboxNotifications/Transformers/ArraySearch.php#L32).

Reply:
Sounds good, thanks!